src/streamlink/stream/ffmpegmux.py

```diff
@@ -91,13 +91,14 @@ def __init__(self, session, *streams, **options):
                              for stream, np in
                              zip(self.streams, self.pipes)]
 
-        ofmt = options.pop("format", "matroska")
+        ofmt = options.pop("format", session.options.get("ffmpeg-fout"))
```

Comment:
the user command **can't work** because `format` is not None

```
$ streamlink https://www.zdf.de/live-tv --ffmpeg-fout "matroska"
[stream.ffmpegmux][debug]

ffmpeg command: /usr/bin/ffmpeg -nostats -y
-i /tmp/ffmpeg-8930-729 -i /tmp/ffmpeg-8930-644
-c:v copy -c:a copy -map 0:v? -map 0:a?
-map 1:a -f mpegts pipe:1
```

same for vcodec acodec start_at_zero

Reply:
Yep, you're right. I have it backwards :/

It should go `ffmpeg-fout` -> `options.format` -> `default`

Reply:
I added some more tests, and defaults in the class - it should behave as expected now.

Reply:
sorry but I see you are using "matroska" for mpegts 
what would be the value to fill in for "flv" ?

Reply:
This issues is ancient, why are you bumping it?